### PR TITLE
clarify that string lengths are utf-16

### PR DESCRIPTION
**Pull request by staltz**
@mycognosist @arj03 @cryptix

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -815,7 +815,7 @@ <h3 id="message-format">Message format</h3>
             </tr>
             <tr>
                 <td>content</td>
-                <td>If the message is not encrypted, This is a dictionary containing free-form data for applications to interpret, plus a mandatory <em>type</em> field. The <em>type</em> field allows applications to filter out message types they don’t understand and must be a string between 3 and 52 characters long (inclusive). If this is a base64 encoded string, followed by a suffiix of <code>.box</code></td>
+                <td>If the message is not encrypted, This is a dictionary containing free-form data for applications to interpret, plus a mandatory <em>type</em> field. The <em>type</em> field allows applications to filter out message types they don’t understand and must be a UTF-16 string between 3 and 52 code units long (inclusive). If this is a base64 encoded string, followed by a suffiix of <code>.box</code></td>
```
Comment (arj03) [inline index.html:818]: Can you fix the suffiiiiiix while you are at it? ;)

Comment (staltz) [reply index.html:818]: oh, lol

Comment (staltz) [reply index.html]: donez

**Comment by arj03** [comment]
Thanks 👍

**Pull request merged**